packages/notifications/push/amplify_push_notifications/pubspec.yaml

```
@@ -0,0 +1,31 @@
+name: amplify_push_notifications
+description: Core Push Notifications Flutter package
+version: 0.0.1
```

```suggestion
version: 0.1.0
```

And update changelog as well